src/actions/actions.ts

```diff
@@ -1960,6 +1960,11 @@ class ActionDeleteChar extends BaseCommand {
 }
 
 @RegisterAction
+class ActionDeleteCharWithDeleteKey extends ActionDeleteChar {
+  keys = ["<delete>"];
+}
```

Comment:
There difference between delete and x is the way they handle numeric prefix. `Nx` will perform `x N times but `N<Del>` does nothing.